pkg/network/node/egressip_test.go

```diff
@@ -64,10 +65,6 @@ type egressOVSChange struct {
 	remote string
 }
 
-// Takes the previous set of egress OVS flows, then fetches the current set and checks
-// that the expected changes have occurred. Each namespace whose egress has changed should
-// have an egressOVSChange struct describing the expected new state. On success, returns
-// the new/current set of flows in flows.
 func assertOVSChanges(eip *egressIPWatcher, flows *[]string, changes ...egressOVSChange) error {
```

Comment:
why did you remove the comment?

Reply:
I thought I would need to make many more changes and have to rewrite the comment but that is not the case